Plugins/GeoModel/include/Acts/Plugins/GeoModel/GeoModelToDetVol.hpp

```diff
@@ -0,0 +1,32 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2024 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#pragma once
+
+#include "Acts/Detector/DetectorVolume.hpp"
+#include "Acts/Detector/GeometryIdGenerator.hpp"
+#include "Acts/Geometry/CuboidVolumeBounds.hpp"
+#include "Acts/Geometry/CutoutCylinderVolumeBounds.hpp"
+#include "Acts/Geometry/CylinderVolumeBounds.hpp"
+#include "Acts/Geometry/GeometryContext.hpp"
+#include "Acts/Geometry/GeometryIdentifier.hpp"
+#include "Acts/Geometry/TrapezoidVolumeBounds.hpp"
+
+#include "GeoModelKernel/GeoShape.h"
+
+namespace Acts {
+namespace GeoModelToDetVol {
```

Comment:
This might be a wider discussion (ping @asalzburger @andiwand @benjaminhuth) but maybe we should standardize our use of namespaces a bit again. 

We originally decided not to add extra namespaces for plugins, but we could start doing this generally.
In that scenario namespaces like this could be mostly replaced with a single `GeoModel` namespace across plugin files, and the function would carry the logical name, like `Acts::GeoModel::convertVolume` or something in that direction.

Reply:
Ok, I would also agree it makes sense to have one namespace across the plugin. I changed the namespace to be GeoModel